### ISSUE TITLE
Rabbits / Bunnies.

### ISSUE THREAD

**Issue by pyrestriker** [Bouncyrock/TaleSpire-Beta-AssetRequests#153]
**Describe the Asset**
Asset is fluffy ball of floof and maybe murder. It's a rabbit.

**Examples (images, videos, something which gets the point across)**
![image](https://user-images.githubusercontent.com/3858960/106344675-1db95600-6260-11eb-9c5a-3ec7d223665e.png)
![image](https://user-images.githubusercontent.com/3858960/106344678-2316a080-6260-11eb-8d94-1c450372f5a9.png)
![image](https://user-images.githubusercontent.com/3858960/106344682-2873eb00-6260-11eb-97c6-d5b144c252dc.png)

**Additional context**
Basically I want to use murderbuns against my players.

<br>**ADD :thumbsup:[thumbs up reaction] to give this request more attention.**

**Comment by pyrestriker** [comment]
Adding to this: an Al-Mi'raj variant. https://en.wikipedia.org/wiki/Al-Mi%27raj
